projects/pureth-eip-7919-nimbus.md

```diff
@@ -0,0 +1,177 @@
+## Pureth [EIP-7919](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7919.md) Implementation for Nimbus-EL
+
+### Collaborators
+
+- [Vineet](https://github.com/vineetpant) 
+- [Tamaghana](https://github.com/RazorClient)
+
+We will coordinate and work parallely on separate EIPs for implementation, once core EIPs (7745 and 6404) are done , we will work on other EIPs for Log Reform and Remove old tx types categories in Pureth.
```

Comment:
Makes sense but this should be part of the roadmap below and make it clear how you dive roles, who works on which EIP

```suggestion
We will coordinate and work in parallel on separate EIPs for implementation, once core EIPs (7745 and 6404) are done , we will work on other EIPs for Log Reform and Remove old tx types categories in Pureth.
```